src/main.rs

```diff
@@ -98,13 +99,9 @@ async fn main() -> Result<()> {
     // adapt the output. This can later be removed if
     // prettytable provides methods to disable color globally
     if no_color {
-        unsafe {
-            env::set_var("TERM", "dumb");
-        }
+        env::set_var("TERM", "dumb");
     } else {
-        unsafe {
-            env::set_var("TERM", term_color_support);
-        }
```

Comment:
The `unsafe` blocks have to stay in order to compile the code with Rust 1.80. At least, building stopped to work as soon as I updated to this version of Rust.

See [here](https://doc.rust-lang.org/std/env/fn.set_var.html#)

Reply:
Oh, I see. Thanks for sharing!  I've removed that because my linter was complaining about it. But I've updated my rust and now the warning/errors goes away